pymc/gp/hsgp_approx.py

```diff
@@ -32,10 +31,13 @@
 
 
 def set_boundary(Xs: TensorLike, c: numbers.Real | TensorLike) -> np.ndarray:
-    """Set the boundary using the `Xs` centered around 0 and `c`. `c` is usually a scalar
-    multiplier greater than 1.0, but it may be one value per dimension or column of `Xs`.
+    """Set the boundary using the `Xs` and `c`.  `Xs` must be centered around zero, and `c`
+    is usually a scalar multiplier greater than 1.0, but it may also be one value per dimension
+    or column of `Xs`.
     """
-    S = pt.max(pt.abs(Xs), axis=0)  # important: the Xs should be centered around 0
+    S = pt.max(
+        pt.abs(Xs), axis=0
+    )  # important: the Xs should have previously been centered around 0
```

Comment:
And if we used the formula for computing the radius instead? It's more generic, so it'll work and give the same results, whether the Xs have been centered or not. That's one possibly confusing place less.
`S = (pt.max(X, axis=0) - pt.min(X, axis=0)) / 2.0`

Reply:
What do you think @bwengals ? After that, I think we're good to merge

Reply:
Just implemented this